source/ProgArgs.h

```diff
@@ -158,6 +158,13 @@ namespace bpt = boost::property_tree;
 #define ARG_S3REGION_LONG			"s3region"
 #define ARG_S3SIGNPAYLOAD_LONG		"s3sign"
 #define ARG_S3TRANSMAN_LONG			"s3transman"
+#define ARG_S3STATDIRS_LONG         "s3statdirs"
+
+#define ARG_S3_BUCKET_TAG_INLINE                "s3btaginline"
+#define ARG_S3_BUCKET_TAG_INLINE_VERIFY         "s3btaginlineverify"
+#define ARG_S3_OBJECT_TAG_INLINE                "s3otaginline"
+#define ARG_S3_OBJECT_TAG_INLINE_VERIFY         "s3otaginlineverify"
```

Comment:
already mentioned in other comment, so just to reference this again here: we need to shorten to max of 16 chars, e.g. s3otaginlverify and s3btaginlverify